### ISSUE TITLE
Error: The alien function "zuncsd_" is undefined

### ISSUE THREAD

**Issue by Roger-luo** [rigetti/quilc#292]
I'm trying to build quilc and run the benchmark, however, after I build the compiler and run with

```
./quilc --benchmark
```

I got

```
! ! ! Error: The alien function "zuncsd_" is undefined.
Benchmarking CL-QUIL-BENCHMARKING::BENCH-0020Q-0000654-JOHANNES-*DEPTH-VS-SWAPS-ASSN*-0020Q-SKEW-RECT...%
```

I'm not very familiar with lisp, anyone could help me fix it? I'm currently building v1.8.2 with `sbcl` 1.5.3 on MacOS.

**Comment by notmgsk** [comment]
@Roger-luo what is the output of the command `./quilc --check-libraries`?

**Comment by Roger-luo** [comment]
```

**Comment by Roger-luo** [comment]
Does this mean it is not correctly linked to lapack? but I didn't see any instruction for linking against lapack... Is there any manual option for this?

**Comment by notmgsk** [comment]
It looks like it is finding an unsupported version of LAPACK. Do you know what version of LAPACK you have? Mine, for example, `3.8.0`.

**Comment by Roger-luo** [comment]
I'm not sure which version it is, but I think it should link to the mkl, which is 2018.3.039 on my laptop. I also got 3.8.0 by openblas in conda, but I guess that's not the lapack it is trying to link to, how can I change the lapack vendor/path for quilc?

**Comment by stylewarning** [comment]
Some versions of LAPACK don't have ZUNCSD unfortunately. This includes MKL and Accelerate I believe. Could you try using a different version of LAPACK?

**Comment by Roger-luo** [comment]
sure, is there a compile option for specific the lapack path then?

**Comment by stylewarning** [comment]
Could you try changing your `LD_LIBRARY_PATH` to the path with the library of interest? A possible problem is that we statically determine the path to these libraries at build time. Are you building the binaries from scratch?

**Comment by rongpenl** [comment]
Hi, any update on this one?

**Comment by jmbr** [comment]
Thank you for reaching out, @rongpenl. Our linear algebra package, magicl was tested a few months ago on several platforms. The CSD decomposition routines work with OpenBLAS on Linux and with the Accelerate framework in macOS. Could you give us some details about your system settings (platform, operating system, available BLAS, versions)?

**Comment by jmbr** [comment]
Closed by #485.
